docs/rule/no-block-params-for-html-elements.md

```diff
@@ -0,0 +1,34 @@
+# no-block-params-for-html-elements
+
+Block parameters (`<Checkbox as |checkbox|>`) are only useful when invoking
+components. When they are used on regular HTML elements they are useless and an
+indicator of a potential bug.
+
+This rule warns about all block parameter usages on regular HTML elements
+(angle bracket invokations with lower-case tagnames).
```

Comment:
```suggestion
(angle bracket invocations with lower-case tagnames).
```

Reply:
how is it "invoke" but "invocation"?! 🙄 😂

Reply:
English is stupid 🤷‍♂️